### DISCUSSION TITLE
Want a fake DS3 controller to be supported.😢

### DISCUSSION THREAD

**Discussion by lkchx123** [nefarius/DsHidMini#130] [Q&A] [answered]
Written in the front: I am a Chinese, this article is translated by Google, there may be some inconsistencies, please understand.😘

I followed the documentation and successfully installed the latest version of DsHidMini and BthPS3. I used Scptoolkit before, and then uninstalled it completely following the documentation. But my controller still does not appear in the list in the DSHMC.exe.😢

![image](https://user-images.githubusercontent.com/46598881/129256698-8a636ee0-5777-42cd-bc89-6be52b368a70.png)

![image](https://user-images.githubusercontent.com/46598881/129257320-2c3d246f-183e-467d-93d9-e8406a70e2cd.png)

As you can see, its name is DS3 Compatible USB Device. Instead of PLAYSTATION(R)3 Controller.😢I add the ‘DS3 Compatible USB Device’ to the list of supported devices on BthPS3 list in Windows Registry. But the same as before.
![image](https://user-images.githubusercontent.com/46598881/129259444-6dbe1daa-5fa4-4d30-be4a-beba5fa95e86.png)

This fake controller can be recognized and work normally with MotioninJoy, ScpToolkit and an Android software Sixaxis Controller. So is there a way to make this controller be recognized by DsHidMini? Is there any way I can try? 

Thanks.😉

**Comment by nefarius** [comment]
That's not how that registry key works whatsoever, that name isn't the device reported name, it's the default from the driver INF, adding it there literally does nothing in your favour.

I will repeat myself until the end of my days: I can and will not waste anymore time on this fake controller debate. I can and will not solve it. Ever. Stop giving scammers your money and stop relying on FOSS to fix other people's shortcomings.

Over and out.

**Reply by lkchx123** [reply]
Sorry, I may have misunderstood the meaning of this sentence.
![image](https://user-images.githubusercontent.com/46598881/129278094-d91f2b95-1b06-4c21-acd8-4d928319baa7.png)
I can fully understand that you don't want to waste time on this kind of thing, and I'm sorry for that.

**Reply by Kanuan** [reply]
Yeah, the project focus changed to "not wasting time with fake controllers that do not work". It's just impossible to propperly identify why they fail to work without:

1. Actually having them and
2. Having a Bluetooth Sniffer that can read the connection between the fake controller and a real PS3

A bluetooth sniffer is too expensive for Nef and he also doesn't have the time anymore to deal with it. Too much work and life stuff to care for random fake controllers. 

Still, glad to see that it's working as normal on your end

**Reply by nefarius** [reply]
I will update that statement, it's no use attracting this topic, thanks for bringing it to my attention.

**Comment by Kanuan** [comment, marked as answer]
It seems the controller is not actually using the DsHidMini driver? It should appear as Nefarius __HID__ device, not USB device.

I think it's loading fireshock, which was the driver used by Shibari? Verify exactly which driver the controller is loading and check the "how to remove conflicting drivers" section of [DsHidMini's install guide](https://vigem.org/projects/DsHidMini/How-to-Install/#fireshock)

![image](https://user-images.githubusercontent.com/24910442/129261017-0e1fa04a-c7ff-40e0-9842-5b6a234a9cbf.png)

**Reply by Kanuan** [reply]
yup, a controller with fireshock will appear on Device Manager under the "Nefarius USB devices" section.

You need to remove the fireshock driver (as well other possible conflicting drivers) in order for the controller to actually load the DsHidMini driver

![image](https://vigem.org/projects/DsHidMini/images/EMS2RXFoc4.png)

**Reply by nefarius** [reply]
Very nice catch, good sir. My remark still holds 😉

**Reply by lkchx123** [reply]
Thank you very much, I didn’t check it completely by myself, and it can work normally now.
